rakuten-f2e/react-redux-homework
# Use `Node.js` & `Express` to apply RESTful design and connect DB for CRUD request

Description
- Use `Node.js` & `Express` to connect to `MongoDB`
- Use `mongoose` & `axios` modules to implement normal RESTful design for CRUD
- Use `babel-node` to let compiler know `ES6` formats

How to test
1. to create a document for db
```
mkdir db
``` 
2. Import fakeData from `./src/client/data/data.js`
```
mongoimport --jsonArray --db crud --collection tabledatas --file <rootDir>/src/client/data/data.json
```
3. Start database
```
mongod --dbpath <rootDir>/db
```
4. Run server on port 3000
```
npm run server
```
5. View in url `localhost:3000`

6. Use shell command `mongo` to see whether we can modify database data successfully
